src/main/java/com/mraof/minestuck/tileentity/redstone/WirelessRedstoneTransmitterTileEntity.java

```diff
@@ -140,11 +161,11 @@ public CompoundNBT save(CompoundNBT compound)
 		
 		compound.putInt("tickCycle", tickCycle);
 		
-		getDestinationBlockPos();
+		getDestinationBlockPosFromOffset();
```

Comment:
From what I can tell, this call here is only there to make sure that `this.offsetPos` is not null (or rather, set to the zero pos if it is null).
I'd suggest instantiating `this.offsetPos` so that it can never be null. Doing so would remove the need of this call and remove the need for a null check in `getDestinationBlockPosFromOffset()`.

Reply:
Upon creation there is a default blockpos set to the variable now